src/test_hmp/wikipedia.py

```diff
@@ -0,0 +1,25 @@
+"""Client for the Wikipedia REST API, version 1.
+
+See `API documentation <https://en.wikipedia.org/api/rest_v1/#/>`_.
+"""
+from typing import Any
+
+import click
+import requests
+
+
+API_URL = "https://{language}.wikipedia.org/api/rest_v1/page/random/summary"
+
+
+def random_page(language: str = "en") -> Any:
```

Comment:
return type is suboptimal but impossible to define Maybe type to handle the exception option